packages/core/src/App/Containers/VerificationModal/verification-modal.scss

```diff
@@ -8,9 +8,18 @@
         height: 100%;
         overflow: auto;
 
+        &__country {
+            &-container {
+                .proof-of-identity__fieldset {
+                    width: 39.5rem;
```

Comment:
I don't think having a fixed width is a good idea for a fieldset, can we make it responsive or follow the container's width?

Reply:
Updated.